### PR TITLE
Update issue templates

### LINKED ISSUE #13: Add GitHub issue templates
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository

### PR DESCRIPTION
**Pull request by tHe-AK**
fixes #13

### CODE REVIEW

**Comment by BaseMax** [comment]
Thanks Akshay.

**Comment by jbampton** [comment]
You need to install `pre-commit`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (jbampton) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]:
```suggestion
 - Browser [e.g. Chrome, Safari]
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
```
Comment (jbampton) [inline .github/ISSUE_TEMPLATE/bug_report.md:34]:
```suggestion
 - Browser [e.g. stock browser, Safari]
```

**Review by jbampton — CHANGES_REQUESTED** [review]

**Review by jbampton — APPROVED** [review]

**Comment by BaseMax** [comment]
In the following:
It will be great, if you can also take a look at:
https://github.com/One-Language/One/tree/master/.github/ISSUE_TEMPLATE

@tHe-AK

**Pull request merged**